acvm-repo/brillig/README.md

```diff
@@ -0,0 +1,237 @@
+# Brillig
+
+## Intro to Brillig
+BRILLIG - Bytecode Runtime Interface for Liberated Language Implementation... G
+
+The G stands for whatever you like, like [Gabagool](https://en.wikipedia.org/wiki/Gabagool).
```

Comment:
```suggestion
```
This was fun when initially written but maybe we remove it now that we will be having auditors reading this 😅 

We can mention at the end of this section after "Brillig is a general virtual machine architecture..." how the name of Brillig is a nonsense word from https://en.wikipedia.org/wiki/Jabberwocky.